src/Data/Singletons/CustomStar.hs

```diff
@@ -51,15 +54,15 @@ import Data.Singletons.Prelude.Bool
 --
 -- generates the following:
 --
--- > data Rep = Nat | Bool | Maybe Rep deriving (Eq, Show, Read)
+-- > data Rep = Nat | Bool | Maybe Rep deriving (Eq, Ord, Read, Show)
 --
 -- and its singleton. However, because @Rep@ is promoted to @*@, the singleton
 -- is perhaps slightly unexpected:
 --
 -- > data instance Sing (a :: *) where
 -- >   SNat :: Sing Nat
 -- >   SBool :: Sing Bool
--- >   SMaybe :: SingRep a => Sing a -> Sing (Maybe a)
+-- >   SMaybe :: Sing a -> Sing (Maybe a)
```

Comment:
I think the old version dates back to version <1.0, when we didn't have the `singInstance` magic.